mdp/nodes/sfa_nodes.py

```diff
@@ -273,8 +273,8 @@ def _stop_training(self, debug=False):
         # not the second central moment matrix (centered about the mean), i.e.
         # the covariance matrix
         if hasattr(self, '_dcov_mtx'):
-            self.dcov_mtx, self.davg, self.dtlen = self._dcov_mtx.fix(
-                center=False)
+            self.dcov_mtx, self.davg, self.dtlen =
+                    self._dcov_mtx.fix(center=False)
```

Comment:
Travis tests fail due to this:

```ImportError while loading conftest '/home/travis/build/mdp-toolkit/mdp-toolkit/mdp/test/conftest.py'.
mdp/__init__.py:171: in <module>
    from . import nodes
mdp/nodes/__init__.py:6: in <module>
    from .sfa_nodes import SFANode, SFA2Node, VartimeSFANode
     File "/home/travis/build/mdp-toolkit/mdp-toolkit/mdp/nodes/sfa_nodes.py", line 276
       self.dcov_mtx, self.davg, self.dtlen =
                                            ^
   SyntaxError: invalid syntax
```

The following `\` could fix it:
```
            self.dcov_mtx, self.davg, self.dtlen =\
                     self._dcov_mtx.fix(center=False)```

Reply:
Ah, sorry, will fix it.